docs/0-Preparing/starting_side_projects.md

```diff
@@ -0,0 +1,93 @@
+
+**You can come up with project ideas**
+
+- *Example*: You noticed that the school newspaper club manually tallies up survey results into different demographics. You write a script that automates this work for them and generates pretty graphs.
+- *Why it's useful*: Employers like to see that you can take initiative.
+
+**You can complete a project on your own**
+
+- *Example*: You make a website to create and send reminders like your parent's birthday. It has user authentication, and a bunch of other thing so other people can use it.
+- *Why it's useful*: It's good to complete a project at some point, as you can learn different things at different stages of a project.
+
+As you can see, the first few don't require you to do anything extraordinary, and can be done even if you've only been programming for two weeks! [See this relevant XKCD](https://xkcd.com/519/).
+
+> You might have heard that the tech industry changes quickly, trends come and go and technologies become obselete quickly. In addition, many companies have proprietary tools internal to the company. That's why it doesn't really matter *what* you learn, as long as you learn *something*. That signals to employers that you can learn whatever is necessary on the job.
+
+### Becoming and staying motivated
+
+It's ideal that you only do side projects if you're enjoying it. That way, it's sustainable, and just a generally healthy thing to do.
+
+But realistically, expect that it can take a few tries to enjoy something. Not all aspects of programming appeal to everyone. It's possible that the stuff you learned in class so far is pretty dry. Some people will enjoy UI/visual/graphics programming. Some people like math and algorithms. Some people like systems. Some people like building products. Some people like to do social good via tech. If you've committed to programming in some way anyway (e.g. enrolled in a CS degree), might as well do some exploration to see what you really like.
+
+### Project ideas
+See this [great GitHub repo](https://github.com/karan/Projects) with tons of project ideas.
+
+**Online courses**
+
+This is often the easiest way to start. You don't need to come up with an idea or even know anything, you can just go around shopping for something that looks interesting. We know, we know, you're taking enough classes at school already. But these courses are often more practical, fun, and have more selection.
+
+They can also leave you with something to show in the end. They allow you to build something super fancy complicated. You can create something through a course (like this [Udacity web development course](https://www.udacity.com/course/web-development--cs253)) or make a game like hangman or build a calculator or whatever else you can think of. You'll be surprised how complicated "simple projects" can be and how much you can learn. That's what employers want to see.
+
+Places to shop for courses: Udacity, Coursera, EdX.
```

Comment:
add something explicitly saying it can be good to find a course that comes along with a project you have completed at the end